addons/api/mirage/factories/scope.js

```diff
@@ -36,6 +36,7 @@ export default factory.extend({
     // Worker permissions only available on the global scope
     if (this.type === 'global') {
       collectionActions.workers = ['create:worker-led', 'list'];
+      collectionActions.aliases = ['create', 'list'];
```

Comment:
Is it global only? Also the comment should change if we add this here

Reply:
yeah it is global only for now, in the future we might extend it to other scopes

Reply:
added